boards/common/nrf52xxxdk/Makefile.include

```diff
@@ -1,6 +1,9 @@
 # this module contains shared code for all boards using the nrf52 CPU
 export CPU = nrf52
 
+# get SoftDevice dependency if needed (dirty hack!)
+include $(RIOTBOARD)/$(BOARD)/Makefile.dep
+
```

Comment:
(this comment got lost in the review I just did)

I (finally) would suggest a message with something like:

The following configuration is dependencies specific but they are resolved later.
Hack to know now if 'nordic_softdevice_ble' is used.

And also, move this include just before using USEPKG.